coral/src/app/features/approvals/schemas/hooks/useTableFilters.tsx

```diff
@@ -20,8 +20,8 @@ const useTableFilters = () => {
   const [searchParams, setSearchParams] = useSearchParams();
 
   const envParam = searchParams.get("environment");
-  const statusParam = searchParams.get("status") as RequestStatus | null;
-  const topicParam = searchParams.get("topic") as string | null;
+  const statusParam = searchParams.get("status") as RequestStatus;
```

Comment:
I thinl `| null` is required here, because that's the value on the first page load :o The returned value of `searchParams.get` is always `string | null`, afaik. 

https://user-images.githubusercontent.com/20607294/221132839-e107afc9-4b12-47df-9207-12e9c4963433.mov


And tbh this cast should be a typeguard later in the code when we use it as part of the request payload, because someone could enter `?status=LOL` in the URL and it would make the API return an error 😬 

![Screenshot 2023-02-24 at 09 45 20](https://user-images.githubusercontent.com/20607294/221133267-16f13aaa-d262-469f-b13a-d3ac005d0b86.png)

Reply:
Updated!